### PR TITLE
repositories: Add picarica overlay

### PR DESCRIPTION
**Pull request by picarica**
i would like to add my overlay :)

not sure if this is the right way, i filled out the template and everything and commited changes

### CODE REVIEW

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -5,59 +5,23 @@
...
-        <email>XXXXX</email>
-        <name>XXXXX</name>
+        <email>dohaje@azet.sk</email>
+        <name>George Bucek</name>
       </owner>
-      <source type="mercurial">https://bitbucket.org/XXXX/xxxx</source>
-      <source type="mercurial">ssh://hg@bitbucket.org/XXXX/xxxx</source>
-      <feed>https://bitbucket.org/XXXX/xxxx/atom</feed>
-      <feed>https://bitbucket.org/XXXX/xxxx/rss</feed>
+      <source type="git">https://gitlab.com/picarica/picarica-overlay.giti</source>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: You seem to have a typo here.

Comment (picarica) [reply files/overlays/repositories.xml]: hello, i fixed all the issues mentioned even the misstype thanks for the notice, i hope its all good now

**Comment by mgorny** [comment]
CI says you don't have correct profiles/repo_name in that overlay.

**Comment by picarica** [comment]
> CI says you don't have correct profiles/repo_name in that overlay.

what do you mean profiles repo name? is my name "picarica" not alright ? or what do you mean by that

**Comment by mgorny** [comment]
You are missing the `profiles/repo_name` file inside the repository.

**Comment by picarica** [comment]
> You are missing the `profiles/repo_name` file inside the repository.

i dont think i followed i entered everything requested name
description
homepage
owner
and source? how do i add the thing you are requesting

**Comment by mgorny** [comment]
You need to add it inside **your overlay**.

**Comment by picarica** [comment]
> You need to add it inside **your overlay**.

ops sorry for misunerstanding its all fixed now hope its okay now

**Comment by mgorny** [comment]
Both names need to be the same, otherwise it's going to get added wrongly and Portage will complain.

**Comment by picarica** [comment]
> Both names need to be the same, otherwise it's going to get added wrongly and Portage will complain.

i hope it all good now

**Comment by picarica** [comment]
is it going to be merged ?

**Pull request closed without merging**